include/wil/resource.h

```diff
@@ -6101,6 +6109,7 @@ typedef unique_any<HPOWERNOTIFY, decltype(&::UnregisterPowerSettingNotification)
 /// @cond
 #define __WIL_PSID_DEF_H_
 /// @endcond
+typedef unique_any<PSID, decltype(&::FreeProcessHeap), ::FreeProcessHeap> unique_process_heap_psid;

```

Comment:
Is this meant to use `wil::details::FreeProcessHeap` (`&details::FreeProcessHeap`)? `FreeProcessHeap` isn't a function in the global namespace

Reply:
Doh! Yes it is. Change to

```
typedef unique_any<PSID, decltype(&::wil::details::FreeProcessHeap), ::wil::details::FreeProcessHeap> unique_process_heap_psid;
```

?

Reply:
From what I can tell that should work. That function is only defined if `winbase.h` is included, which AFAICT is a prerequisite to having `__WIL_WINBASE_DESKTOP` defined.

FYI - everyone should have permission & access to see pipeline results. It can sometimes be a pain finding the specific links you need to click to eventually browse to ADO, but it should be there. E.g.:
https://dev.azure.com/msft-wil/Windows%20Implementation%20Library/_build/results?buildId=1090&view=logs&j=e4143947-9299-5fc1-925b-b37a5c5aa01c&t=a76536c3-2b16-5b67-febe-f59dccb09a0a

Reply:
I went with `details::FreeProcessHeap` rather than `::wil::details::FreeProcessHeap` given precedent (`unique_process_heap_string`)